atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraTimestampStore.java

```diff
@@ -112,33 +143,54 @@ public synchronized void backupExistingTimestamp() {
      * This may be implemented following the inverse of the backup process:
      *  - Read the value of the timestamp. If this is readable, skip.
      *  - Read the value of the backup (BT). If this is unreadable, fail.
-     *  - Do a conditional logged batch:
```

Comment:
I've clarified some of the less obvious but intentional semantics, though, e.g. multiple calls to backup will all return the backed up value.